airbyte-integrations/connectors/source-recharge/source_recharge/api.py

```diff
@@ -79,20 +84,6 @@ def should_retry(self, response: requests.Response) -> bool:
 
         return super().should_retry(response)
 
-    def stream_slices(
```

Comment:
what's the reason for removing slicing?

Reply:
now connector uses `next_cursor` from api response to get next data, and first request uses start date from config or from stream state, so we don't need stream slices at all.

Reply:
Stream slicing enables us use checkpointing, therefore I suggest you leave stream slicing as is or modify if needed, but do not remove it. Use pagination inside the slice please.